src/Agent.Worker/Release/ReleaseServer.cs

```diff
@@ -5,41 +5,69 @@
 using System.Collections.Generic;
 using System.Threading;
 using System.Threading.Tasks;
-
 using Microsoft.VisualStudio.Services.Agent.Util;
-using Microsoft.VisualStudio.Services.Common;
 using Microsoft.VisualStudio.Services.ReleaseManagement.WebApi.Clients;
 using Microsoft.VisualStudio.Services.ReleaseManagement.WebApi.Contracts;
 using Microsoft.VisualStudio.Services.WebApi;
 using RMContracts = Microsoft.VisualStudio.Services.ReleaseManagement.WebApi;
 
-namespace Agent.Worker.Release
+namespace Microsoft.VisualStudio.Services.Agent.Worker.Release
 {
-    public class ReleaseServer
+    [ServiceLocator(Default = typeof(ReleaseServer))]
+    public interface IReleaseServer : IAgentService
```

Comment:
We will need to test releases to make sure none of these changes break something in RM.